util/util.go

```diff
@@ -58,3 +63,50 @@ func AddImageNames(store storage.Store, image *storage.Image, addNames []string)
 	}
 	return nil
 }
+
```

Comment:
FWIW, the code below was "borrowed" from skopeo.  I believe the only change I made to is was to uppercase the first character in contextFromGlobalOptions and a couple of comments from Dan for error formatting..